pandas/_libs/__init__.py

```diff
@@ -2,4 +2,4 @@
 # flake8: noqa
 
 from .tslibs import (
-    iNaT, NaT, Timestamp, Timedelta, OutOfBoundsDatetime, Period)
+    iNaT, NaT, NaTType, Timestamp, Timedelta, OutOfBoundsDatetime, Period)
```

Comment:
Not sure if there's any objection to doing this, but if we didn't want to expose this would need to update type comments @jbrockmendel

Reply:
whats the motivation here?  I wouldn't want to expose this ideally.  But its explicitly private so not really harmful.